pkg/sql/row/fetcher.go

```diff
@@ -742,6 +742,7 @@ func (rf *Fetcher) ReadIndexKey(
 ) (remaining []byte, ok bool, foundNull bool, err error) {
 	// If there is only one table to check keys for, there is no need
 	// to go through the equivalence signature checks.
+
```

Comment:
[nit]: is this needed?

Reply:
leftover from a whole lot of debug print statements ;) removing.